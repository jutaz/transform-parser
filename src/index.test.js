const parser = require('../src');

describe('parser', () => {
  test('should parse multiple operations', () => {
    expect(parser.parse(
      'translate(2508.000000, 32.000000) scale(-1, 1) translate(-2508.000000, -32.000000) translate(2482.000000, 1.000000)'
    )).toMatchSnapshot()
  })
})
