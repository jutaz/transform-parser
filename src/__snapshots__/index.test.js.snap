// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parser should parse multiple operations 1`] = `
Array [
  Object {
    "type": "translate",
    "x": Object {
      "unit": null,
      "value": 2508,
    },
    "y": Object {
      "unit": null,
      "value": 32,
    },
    "z": null,
  },
  Object {
    "type": "scale",
    "x": -1,
    "y": 1,
    "z": null,
  },
  Object {
    "type": "translate",
    "x": Object {
      "unit": null,
      "value": -2508,
    },
    "y": Object {
      "unit": null,
      "value": -32,
    },
    "z": null,
  },
  Object {
    "type": "translate",
    "x": Object {
      "unit": null,
      "value": 2482,
    },
    "y": Object {
      "unit": null,
      "value": 1,
    },
    "z": null,
  },
]
`;
